pycode/memilio-surrogatemodel/memilio/surrogatemodel/ode_secir_simple/network_architectures.py

```diff
@@ -30,51 +34,50 @@ def multilayer_multi_input():
     return model
 
 
-def lstm_network_multi_input():
+def lstm_network_multi_input_single_output():
+    """! LSTM Network which uses multiple time steps as input and returns the 8 compartments for one single time step in the future.
+
+    Input and output have shape [batch, time, features].
```

Comment:
```suggestion
    Input and output have shape [number of expert model simulations, time points in simulation, number of individuals in infection states].
```

Can we be more precise on this? Is this correct? Also applies below.